src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -66,19 +68,15 @@ internal void LogDiagnostic(Diagnostic diagnostic, CultureInfo culture)
                 message = "<None>";
             }
 
-            string description = diagnostic.Descriptor.Description.ToString(culture);
-            if (string.IsNullOrEmpty(description))
-            {
-                _writer.Write("fullMessage", message);
-            }
-            else
+            _writer.Write("message", message);
+
+            if (diagnostic.IsSuppressed)
             {
-                _writer.Write("shortMessage", message);
-                _writer.Write("fullMessage", description);
+                _writer.WriteArrayStart("suppressionStates");
+                _writer.Write("suppressedInSource");
+                _writer.WriteArrayEnd();
             }
 
-            _writer.Write("isSuppressedInSource", diagnostic.IsSuppressed);
-
             WriteTags(diagnostic);
```

Comment:
Do you know about the tags change we made? We now exclusively have a 'properties' object that holds all tool-specific stuff. This property bag has a reserved key name, 'tags', that must refer to a string array, if present. So, to emit a tag 'blah' do this:

"properties" : { "tags" : ["blah"], ... other props here ... }